selfdrive/car/ford/values.py

```diff
@@ -79,6 +84,10 @@ class FordCarInfo(CarInfo):
   def init_make(self, CP: car.CarParams):
     if CP.carFingerprint in (CAR.BRONCO_SPORT_MK1, CAR.MAVERICK_MK1):
       self.car_parts = CarParts([Device.threex_angled_mount, CarHarness.ford_q3])
+    if CP.carFingerprint in (CAR.F_150_LIGHTNING_MK1):
+      self.car_parts = CarParts([Device.threex_angled_mount, CarHarness.ford_q4])
+    if CP.carFingerprint in (CAR.MUSTANG_MACH_E_MK1):
+      self.car_parts = CarParts([CarHarness.ford_q4])
```

Comment:
Looks like we don't set Q4 for F150?! Let's just check if CANFD car to add the q4 harness. Also probably want to add angled mount to ICE F150, no?